packages/venia-ui/lib/components/AuthBar/authBar.js

```diff
@@ -1,43 +1,57 @@
 import React from 'react';
 import { bool, func, shape, string } from 'prop-types';
+import { ArrowRight as ArrowRightIcon } from 'react-feather';
+
+import { useAuthBar } from '@magento/peregrine/lib/talons/AuthBar/useAuthBar';
 
 import { mergeClasses } from '../../classify';
-import Button from '../Button';
-import UserChip from './userChip';
+import AccountChip from '../AccountChip';
+import Icon from '../Icon';
 import defaultClasses from './authBar.css';
-import { useAuthBar } from '@magento/peregrine/lib/talons/AuthBar/useAuthBar';
 
 const AuthBar = props => {
-    const classes = mergeClasses(defaultClasses, props.classes);
-
     const {
-        currentUser,
         handleShowMyAccount,
         handleSignIn,
-        isSignedIn,
+        isUserSignedIn,
         isSignInDisabled
     } = useAuthBar(props);
 
-    const child = isSignedIn ? (
-        <UserChip user={currentUser} showMyAccount={handleShowMyAccount} />
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const handleClick = isUserSignedIn ? handleShowMyAccount : handleSignIn;
```

Comment:
How would this compare against having 2 different buttons for signing and my account based on `isUserSignedIn`? I feel having different buttons gives more control over its style and interactions.

Reply:
Resolved by https://github.com/magento/pwa-studio/commit/cd13df1f763517d741c1b163337bb3156bdb9d41.